docker/build-uc/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM eclipse-temurin:17-jdk-alpine AS build_uc
+
+# Install any required packages
+RUN <<EOF
+    set -ex;
+    apk update;
+    apk upgrade; 
+    apk add bash git;
+EOF
+
+# Define the shell used within the container
+SHELL ["/bin/bash", "-i", "-c", "-o", "pipefail"] 
+
+ARG unitycatalog_home="/opt/unitycatalog"
+ARG unitycatalog_repo="${unitycatalog_home}/repo"
+ARG unitycatalog_github="https://github.com/unitycatalog/unitycatalog.git"
+# Specify which version to pull ex.: 0.1.0
+ARG unitycatalog_version="main"
+# Specify any custom parameters necessary to generate
+# the Uber-Jar by SBT. 
+# Note: The default allocated heam memory size is too small
+# and will cause the process to fail when attempting to compile
+# and generate the Uber-Jar. Therefore it is important to choose
+# a size large enough for the compiler to run. 
+ARG sbt_args="-J-Xmx5G"
```

Comment:
I would keep packaging and testing as a separate step. How about adding:

```
  assembly / test := {},
```
to `build.sbt`?